scripts/start_with_docker_develop.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
These scripts should be added to instructions posted here:
https://yaptide.github.io/yaptide/using_docker/
but that can be done later in separate issue.